zipline/pipeline/engine.py

```diff
@@ -82,9 +82,8 @@ def run_chunked_pipeline(self, pipeline, start_date, end_date, chunksize):
             The start date to run the pipeline for.
         end_date : pd.Timestamp
             The end date to run the pipeline for.
-        chunksize : int or None
-            The number of days to execute at a time. If None, then
-            results will be calculated for entire date range at once.
+        chunksize : int
+            The number of days to execute at a time.
```

Comment:
If a user is going to explicitly pass None, they should probably just use `run_pipeline`.